projects/canopy/src/lib/carousel/carousel-item/carousel-item.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="lg-carousel-item" [ngClass]="'lg-padding--' + paddingSize">
```

Comment:
One convention we try and follow on Canopy is to use the wrapping element that Angular creates. So in this instance we would probably do...

```ts
@HostBinding('class.lg-carousel-item') class = true;
```

And then your template would simply be
```html
<ng-content></ng-content>
```

Meaning the final markup is..
```html
<lg-carousel-item class="lg-carousel-item">
  content
</lg-carousel-item>
```
as opposed to the slightly heavier...
```html
<lg-carousel-item>
  <div class="lg-carousel-item">
    content
  </div>
</lg-carousel-item>
```

I also wondered about removing the padding options altogether, if we provide a sensible default this could be overridden using the [padding](https://deploy-preview-386--legal-and-general-canopy.netlify.app/?path=/story/directives--padding) or [margin](https://deploy-preview-386--legal-and-general-canopy.netlify.app/?path=/story/directives--margin) directives